nextjs/pages/index.tsx

```diff
@@ -28,7 +28,7 @@ function App() {
     const client = createPromiseClient(
         ElizaService,
         createConnectTransport({
-            baseUrl: 'https://demo.connect.build',
+            baseUrl: 'http://localhost:3000/api',
```

Comment:
To make this work when publishing on vercel without modifying the source just to deploy:

```suggestion
            baseUrl: '/api',
```